### DISCUSSION TITLE
SARIF file limits on runs

### DISCUSSION THREAD

**Discussion by RorySimpson** [community/community#30409] [Actions]
Hello,

As part of our CI using Github Actions, we are using Snyk to generate a security report in a SARIF file format and then attempting to upload this SARIF File to Github Code Scanning. The SARIF file is failing to upload due to the below error:
```
Error: Code Scanning could not process the submitted SARIF file:
  rejecting SARIF, as there are more runs than allowed (25 > 15)
```

Is this expected behaviour as we can't find the limit for the number of runs in a SARIF file documented anywhere? Also is there anyway for this limit to be configured higher?

**Comment by simon-engledew** [comment]
👋 Hey,

This is the expected behaviour as SARIF files with many runs can indicate a configuration issue. It might be worth reaching out to Synk to see if they can suggest a configuration that generates a smaller number of runs. 🙏
